app/routes/datasets/queries.py

```diff
@@ -646,8 +646,8 @@ def _get_area_density_name(nm):
     return nm with the area-density suffix removed."""
     for suffix in AREA_DENSITY_RASTER_SUFFIXES:
         if nm.endswith(suffix):
```

Comment:
What nm mean again?

Reply:
nm is the original pixel_meaning or source_layer_name before the possible magic suffix is removed.